content/es/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,67 @@
+
+El nombre de recurso de un objeto LimitRange debe ser un
+[nombre de subdominio DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) válido.
+
+### Aplicando Límites de rango
+
+- El administrador crea un LimitRange en un {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
+- Los usuarios crean recursos como {{< glossary_tooltip text="Pods" term_id="pod" >}}, {{< glossary_tooltip text="contenedores" term_id="container" >}} o {{< glossary_tooltip text="peticiones de volúmenes persistentes" term_id="persistent-volume-claim" >}} en el {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
+- El controlador de admisión `LimitRanger` aplicará valores predeterminados y límites, para todos los {{< glossary_tooltip text="Pods" term_id="pod" >}} o {{< glossary_tooltip text="Contenedores" term_id="container" >}} que no establezcan requisitos de recursos informáticos. Y realiza un seguimiento del uso para garantizar que no excedan el mínimo, el máximo, y la proporción de ningún LimitRange definido en el {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
+- Si al crear o actualizar un recurso del ejemplo ({{< glossary_tooltip text="Pods" term_id="pod" >}}, {{< glossary_tooltip text="contenedores" term_id="container" >}}, {{< glossary_tooltip text="peticiones de volúmenes persistentes" term_id="persistent-volume-claim" >}}) se viola una restricción al LimitRange, la solicitud al servidor API fallará con un código de estado HTTP "403 FORBIDDEN" y un mensaje que explica la restricción que se ha violado.
+- En caso de que en se active un LimitRange para recursos de cómputos como `cpu` y `memory`, los usuarios deberán especificar las solicitudes o límites de recursos a dichos valores. De lo contrario, el sistema puede rechazar la creación del {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+- Las validaciones de LimitRange ocurren solo en la etapa de Admisión de Pod, no en {{< glossary_tooltip text="Pods" term_id="pod" >}} que ya se han iniciado (Running {{< glossary_tooltip text="Pods" term_id="pod" >}}).
+
+Algunos ejemplos de políticas que se pueden crear utilizando rangos de límites son:
+
+- En un clúster de 2 nodos con una capacidad de 8 GiB de RAM y 16 núcleos, podría restringirse los {{< glossary_tooltip text="Pods" term_id="pod" >}} en un {{< glossary_tooltip text="Namespace" term_id="namespace" >}} a solicitar `100m` de CPU con un límite máximo de `500m` para CPU y solicitar `200Mi` de memoria con un límite máximo de `600Mi` de memoria.
+- Definir el valor por defecto de límite y requisitos de CPU a `150m` y el valor por defecto de solicitud de memoria a `300Mi` {{< glossary_tooltip text="contenedores" term_id="container" >}} que se iniciaron sin requisitos de CPU y memoria en sus especificaciones.
```

Comment:
```suggestion
- Definir el valor por defecto de límite y petición de CPU a `150m` y el valor por defecto de petición de memoria a `300Mi` para {{< glossary_tooltip text="contenedores" term_id="container" >}} que se iniciaron sin solicitud de CPU y memoria en sus especificaciones.
```

Reply:
No considero apropiado utilizar **petición**, no porque esté mal, sino porque esa palabra se relaciona más con la acción/proceso de solicitar los recursos que con la documentación de resources.requests

#### [Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#resourcerequirements-v1-core)

A propósito, las sugerencias que hiciste fueron tomadas y los cambios realizados.
* Voy a proceder a resolver esta sugerencia,
* **luego hacer un nuevo commit para corregir un **weight** que cambié por error**  **# _important!_**
* y luego podrías volver a hacer la sugerencia si te parece necesario. 

(creo que no haber resuelto la sugerencia fue lo que impidió el merge automático de la sugerencia, o quizá porque hice los cambio por una petición via mensaje)

Reply:
@electrocucaracha perdón que resolví la conversación tan abruptamente. Le estoy agarrando la mano y ahora entiendo por qué no me era posible agregar cambios sugeridos (porque había Resuelto la conversación.. que lío me hice).
Vuelvo a abrir la discusión para saber tu parecer.
Saludos